emails/models.py

```diff
@@ -137,36 +184,35 @@ def __str__(self):
     def delete(self, *args, **kwargs):
         # TODO: create hard bounce receipt rule in AWS for the address
         deleted_address = DeletedAddress.objects.create(
-            address_hash=sha256(self.address.encode('utf-8')).hexdigest(),
+            address_hash=address_hash(self.address),
             num_forwarded=self.num_forwarded,
             num_blocked=self.num_blocked,
             num_spam=self.num_spam,
         )
         deleted_address.save()
         profile = Profile.objects.get(user=self.user)
-        profile.address_last_deleted = datetime.now()
+        profile.address_last_deleted = datetime.now(timezone.utc)
```

Comment:
@groovecoder added the timezone here because it was complaining on tests AND because we have `USE_TZ = True` so we should include timezone.